src/angular/toggle/toggle.scss

```diff
@@ -0,0 +1,194 @@
+@import '../styles/common';
```

Comment:
general design issues:
 - option in lean design has doubled borders and rounded corners between
 - the triple variant in standard design has not aligned content borders with the right toggle border of the third toggle